crates/rolldown/tests/fixtures/misc/outro/iife/artifacts.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/fixtures/misc/outro/iife
+---
+# Assets
+
+## main.mjs
+
+```js
+(function() {
+
+"use strict";
+
+//#region main.js
+var main_default = "banner";
+
+//#endregion
+return main_default;
+console.log('hello world');
```

Comment:
The iife `outer` shoule be ahead at `return`. see [here](https://rollupjs.org/repl/?version=4.19.0&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQW51bGwlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIyY29kZSUyMiUzQSUyMiUyRiUyRiUyMFRSRUUtU0hBS0lORyU1Q25leHBvcnQlMjBjb25zdCUyMHZhbHVlJTIwJTNEJTIwMSUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTJDJTIybmFtZSUyMiUzQSUyMm1haW4uanMlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyb3V0cHV0JTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyaWlmZSUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMm91dHJvJTIyJTNBJTIyJTJGJTJGJTJGJTIwYWElMjIlN0QlMkMlMjJ0cmVlc2hha2UlMjIlM0F0cnVlJTdEJTdE)

Reply:
Only in `output.exports: 'named'` and in `iife` should the `outer` being ahead of the `return`. Check it out here: [`iife` `default`](https://rollupjs.org/repl/?version=4.19.0&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQW51bGwlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIyY29kZSUyMiUzQSUyMiUyRiUyRiUyMFRSRUUtU0hBS0lORyU1Q25leHBvcnQlMjBkZWZhdWx0JTIwJTVDJTIyaGVsbG8lNUMlMjIlMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSUyQyUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMm91dHB1dCUyMiUzQSU3QiUyMmV4cG9ydHMlMjIlM0ElMjJkZWZhdWx0JTIyJTJDJTIyZm9ybWF0JTIyJTNBJTIyaWlmZSUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMm91dHJvJTIyJTNBJTIyJTJGJTJGJTJGJTIwYWElMjIlN0QlMkMlMjJ0cmVlc2hha2UlMjIlM0F0cnVlJTdEJTdE), and IIFE `named` in your example.